fuelclient/cli/actions/network.py

```diff
@@ -58,6 +79,38 @@ def upload(self, params):
         print(
             "Network configuration uploaded."
         )
+        if params.dnsmasq:
+            self._update_dnsmasq_template(network_data['networks'])
+
+    @master_only
+    def _update_dnsmasq_template(self, networks):
+        fw_admin_nets = dict([(n['group_id'], n) for n in networks if
+                              n['name'] == 'fuelweb_admin'])
+        base_net = fw_admin_nets.pop(None)
+        for group_id, net in fw_admin_nets.iteritems():
+            marker = GROUP_ID_TEMPLATE.format(group_id = group_id)
+            try:
+                exec_cmd(COBBLER_SHELL + 'grep -q "%s" %s' % (
```

Comment:
I would make "exec_cmd_on_cobbler" or "exec_cmd_on_container" (depending how much you want to make it reusable)